### ISSUE TITLE
Feature request: enhance commentSearch() to allow showing only removed comments

### ISSUE THREAD

**Issue by jpettitt** [creesch/reddit-moderator-toolbox#794]
In one of my high traffic subs we're looking at a new anti troll policy where if a threshold number of substantiated reports is met we ban the user for 30 days.  It would be really nice to be able to list all the removed comment a user has made so we can get a quick snapshot of how much of an ass they are.  

In a perfect world there would be a one click removal % stat right on the post/comment ...

Possible? Difficult?

**Comment by creesch** [comment]
Yeah should be possible. Though when dealing with comments and people doing a lot of it you run into the  1000 item limit rather quickly. 

So we probably will build it in at some point, but for your usecase you might want to actually pull the entire modlog and search for removed comments by that user there. 

In fact, if we implement this it will probably be through the modlog as that makes more sense and goes back further.
